### PR TITLE
replace django.core.urlresolvers with django.urls

### PR DESCRIPTION
**Pull request by nickpell**
remove deprecation warnings like:

RemovedInDjango20Warning: Importing from django.core.urlresolvers is deprecated in favor of django.urls.
  from django.core.urlresolvers import reverse

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/models.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 from django.core.cache import cache
 from django.conf import settings
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/models.py]: F401 'reverse' imported but unused

File: custom/_legacy/pact/reports/chw_list.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from django.core.urlresolvers import  NoReverseMatch
+from django.urls import  NoReverseMatch
```
Comment (dimagimon) [inline custom/_legacy/pact/reports/chw_list.py]: E271 multiple spaces after keyword

File: custom/_legacy/pact/reports/patient_list.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from django.core.urlresolvers import  NoReverseMatch
+from django.urls import  NoReverseMatch
```
Comment (dimagimon) [inline custom/_legacy/pact/reports/patient_list.py]: E271 multiple spaces after keyword

**Comment by nickpell** [comment]
@esoergel

**Review by esoergel — APPROVED** [review]

**Pull request merged**